### DISCUSSION TITLE
Student status verification loop

### DISCUSSION THREAD

**Discussion by jaedm97** [community/community#65611] [GitHub Education]
### Select Topic Area

Question

### Body

Dear concern,

When I go to reapply for the Student Developer Pack I get this error message -

`Hi, Jaed Mosharraf! You were last verified as a student on. It is not necessary for you to re-verify at this time.`

And my pro badge is also hidden now. Could anyone please help me with this? I am stuck and in a loop for getting it.

Regards

**Comment by 79993249** [comment]
same issue

**Comment by matejnevlud** [comment]
same here
